analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/PermissiveCorsTest.cs

```diff
@@ -35,27 +36,33 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class PermissiveCorsTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder()
```

Comment:
- There is one more occurrence to change at line 84.
- I would move `.AddReferences(AdditionalReferences)` here since it's used everywhere.